# Written internally- removing -Wall flags because Solaris Studio does not know of them
# Since this is for our build, probably not something we can get upstream.

--- ORIGINAL/src/Makefile.am	2016-01-12 08:43:52.928382761 -0800
+++ gparted-0.4.5/src/Makefile.am	2016-01-12 08:46:21.389297831 -0800
@@ -3,10 +3,6 @@
 	-DGPARTED_DATADIR=\""$(datadir)"\"			\
 	-DGNOMELOCALEDIR=\""$(datadir)/locale"\"
 
-AM_CFLAGS = -Wall	
-
-AM_CXXFLAGS = -Wall	
-
 sbin_PROGRAMS = gpartedbin
 
 gpartedbin_SOURCES = \
